packages/arb-bridge-eth/contracts/inbox/GlobalFTWallet.sol

```diff
@@ -33,6 +34,15 @@ contract GlobalFTWallet {
 
     mapping(address => UserFTWallet) private ftWallets;
 
+    enum PairingStatus { Unpaired, Requested, Paired }
```

Comment:
If this is implicitly relying on Unpaired being mapped to zero, so that uninitialized items are implicitly Unpaired, this should be noted in a comment.